src/azure-cli/azure/cli/command_modules/configure/_help.py

```diff
@@ -53,10 +53,30 @@
 
 helps['local-context on'] = """
 type: command
-short-summary: Turn on local context in current directory
+short-summary: Turn on local context
```

Comment:
Let's add

> The context information is saved in the current working directory

Reply:
I think it may be confused to user if add this. It's because the information is not actually saved in the working directory where user turn it on. If the user turn it on in dir1 and then switch to dir2, and run a command which support local context, the information will be saved in dir2. So I prefer not to add it in `az local-context on` command.